corehq/warehouse/etl.py

```diff
@@ -125,3 +113,12 @@ def _render_template(path, context):
 
     template = engines['django'].from_string(template_string)
     return template.render(context)
+
+
```

Comment:
should this be `memoized`? i think it depends on the `slugs` that would not change unless there is a change in code.

Reply:
it could be but I don't think it's worth it since it's a cheap operation and it doesn't get called frequently